proposed/2021/NuGetConsumePDBInPackageReference.md

```diff
@@ -0,0 +1,216 @@
+# Consuming pdb support with PackageReference
+
+* Status: Reviewing
+* Author(s): [Heng Liu](https://github.com/heng-liu)
+* Issue: [5926](https://github.com/NuGet/Home/issues/5926) Consume pdbs from packages in PackageReference - add Debug Symbols to LockFileTargetLibrary
+* Type: Feature
+
+## Problem Background
+
+In the PackageReference world, the packages are consumed from the global-packages folder, .pdb, .xml files from lib and runtime folder are not copied into the build output folder.
+In some cases, consuming .pbd, .xml files are needed to support certain features, like source link ( see issue [1458](https://github.com/dotnet/sdk/issues/1458)).
+The established design for consuming files out of packages is that NuGet tells the .NET SDK which files are important through the assets file, or to be specific, the [LockFileTargetLibrary](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.ProjectModel/LockFile/LockFileTargetLibrary.cs). 
+Currently, NuGet does not select pdb or xml files.
+
+In comparison, in packages.config, .pdb, .xml files are automatically copied from lib and runtime folder into the build output folder because of the [ResolveAssemblyReference](https://github.com/dotnet/msbuild/blob/main/src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs#L75) task. So there is no such problem in packages.config.
```

Comment:
nit: The runtime folder is a PackageReference construct, and not a thing in packages.config.

Reply:
Good to know! Removed runtime folder.